cli/emit.rs

```diff
@@ -192,7 +192,8 @@ pub fn get_ts_config(
         ts_config.merge(&json!({
           "emitDecoratorMetadata": false,
           "importsNotUsedAsValues": "remove",
-          "inlineSourceMap": true,
+          "inlineSourceMap": false,
+          "sourceMap": true,
```

Comment:
I'm a bit confused - from my understanding this should only have an impact on TSC emit for `deno` subcommand and `Deno.emit()` API shouldn't be altered at all and yet tests that use `Deno.emit()` started failing.